docs/reference/run.md

```diff
@@ -149,7 +150,7 @@ is receiving its standard input from a pipe, as in:
 The operator can identify a container in three ways:
 
 | Identifier type       | Example value                                                      |
-| --------------------- | ------------------------------------------------------------------ |
+|:----------------------|:-------------------------------------------------------------------|
```

Comment:
Looks like it's the `left align` (`|:-` making the difference)

And aligning the right-hand border for the other tables